.github/workflows/main.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       - name: Setup Deno
         uses: denoland/setup-deno@v1
         with:
-          deno-version: "^1.0.0"
+          deno-version: "1.18.2"
```

Comment:
There is another section in this file where we need to change the version on line 151

Reply:
@mweichert

Reply:
Updated.